{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"additionalErrorHandler"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo\/ApolloErrorInterceptor","preciseIdentifier":"s:6Apollo0A16ErrorInterceptorP","text":"ApolloErrorInterceptor"},{"kind":"text","text":"?"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/apollo\/requestchain\/additionalerrorhandler"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo\/RequestChain\/additionalErrorHandler","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Something which allows additional error handling to occur when some kind of error has happened."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"additionalErrorHandler"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"ApolloErrorInterceptor","preciseIdentifier":"s:6Apollo0A16ErrorInterceptorP"},{"kind":"text","text":"?"}],"title":"additionalErrorHandler","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:6Apollo12RequestChainC22additionalErrorHandlerAA0aE11Interceptor_pSgvp","modules":[{"name":"Apollo"}]},"hierarchy":{"paths":[["doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo","doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo\/RequestChain"]]},"references":{"doc://com.apollographql.Apollo.macosx/documentation/Apollo":{"role":"collection","title":"Apollo","abstract":[],"identifier":"doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo","kind":"symbol","type":"topic","url":"\/documentation\/apollo"},"doc://com.apollographql.Apollo.macosx/documentation/Apollo/RequestChain":{"role":"symbol","title":"RequestChain","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"RequestChain"}],"abstract":[{"type":"text","text":"A chain that allows a single network request to be created and executed."}],"identifier":"doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo\/RequestChain","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"RequestChain"}],"url":"\/documentation\/apollo\/requestchain"},"doc://com.apollographql.Apollo.macosx/documentation/Apollo/RequestChain/additionalErrorHandler":{"role":"symbol","title":"additionalErrorHandler","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"additionalErrorHandler"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"ApolloErrorInterceptor","preciseIdentifier":"s:6Apollo0A16ErrorInterceptorP"},{"kind":"text","text":"?"}],"abstract":[{"type":"text","text":"Something which allows additional error handling to occur when some kind of error has happened."}],"identifier":"doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo\/RequestChain\/additionalErrorHandler","kind":"symbol","type":"topic","url":"\/documentation\/apollo\/requestchain\/additionalerrorhandler"},"doc://com.apollographql.Apollo.macosx/documentation/Apollo/ApolloErrorInterceptor":{"role":"symbol","title":"ApolloErrorInterceptor","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"ApolloErrorInterceptor"}],"abstract":[{"type":"text","text":"An error interceptor called to allow further examination of error data when an error occurs in the chain."}],"identifier":"doc:\/\/com.apollographql.Apollo.macosx\/documentation\/Apollo\/ApolloErrorInterceptor","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ApolloErrorInterceptor"}],"url":"\/documentation\/apollo\/apolloerrorinterceptor"}}}